doc/api/deprecations.md

```diff
@@ -3212,7 +3215,7 @@ changes:
 
 Type: Runtime
 
-The `'hash'` and `'mgf1Hash'` options are replaced with `'hashAlgorithm'`
+The `'hash'` and `'mgf1Hash'` options were replaced with `'hashAlgorithm'`
 and `'mgf1HashAlgorithm'`.
```

Comment:
nit: it might be clearer to phrase it as a migration guide
```suggestion
Use  `'hashAlgorithm'` instead of `'hash'`, and `'mgf1HashAlgorithm'` instead of `'mgf1Hash'`.
```